src/compiler/scala/tools/nsc/typechecker/Implicits.scala

```diff
@@ -572,6 +572,40 @@ trait Implicits {
        }
      }
 
+    private def matchesPtInst(info: ImplicitInfo): Boolean = {
+      // the pt for views can have embedded unification type variables or BoundedWildcardTypes which
+      // can't be solved for. Rather than attempt to patch things up later we just skip those cases
+      // altogether.
+      def isInstantiable =
+        !isView || (wildPt.isGround && !wildPt.exists { case _: BoundedWildcardType => true ; case _ => false })
```

Comment:
I'd pull out the `isInstantiable` check to the `matchesPtInst` call below, and cache whether `wildPt` contains `TypeVar`s (`!isGround`) or `BoundedWildcardType`s. I believe that test could be written more directly as `!wildPt.exists { case _: BoundedWildcardType | _: TypeVar => true ; case _ => false }`.